charts/airflow/CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ This project follows [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
 TBD
 
+## [8.6.0] - 2021-08-25
```

Comment:
Please don't update the changelog, we only do this in the "release commit"

Reply:
Please remove all updates to the changelog (we only do this for commits that trigger releases).